cmd/soroban-rpc/internal/test/get_ledger_entry_test.go

```diff
@@ -0,0 +1,150 @@
+package test
```

Comment:
just wanted to mention, I added this coverage scope as a system test(dapp e2e) also:
https://github.com/stellar/system-test/issues/3, perhaps at some point can consolidate integration tests in there.